### PR TITLE
docs: add shoutify.app to t3 collection

### PR DESCRIPTION
**Pull request by KyleTryon**
Closes https://github.com/TechSquidTV/Shoutify/issues/40

## ✅ Checklist

- [ x ] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-08-15).
- [ x ] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ x ] I performed a functional test on my final commit

---

## Changelog

add shoutify.app to t3 collection. 

---

The link given for the website is currently dead, but we will get a landing page there as soon as possible.

### CODE REVIEW

**Comment by nexxeln** [comment]
Run `pnpm format` and push.

File: www/src/pages/en/t3-collection.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Made a project using the T3 stack and want to share it? Add it to the list!
 | Description                                                      | Repo                                                                  | Link                               |
 | ---------------------------------------------------------------- | --------------------------------------------------------------------- | ---------------------------------- |
 | Zapdos - a QnA app for streamers                                 | [t3-oss/zapdos](https://github.com/t3-oss/zapdos)                     | [z.t3.gg](https://z.t3.gg)         |
+| Shoutify - An open source, self-hosted, and entirely free solution to social media management.                                 | [techsquidtv/shoutify](https://github.com/TechSquidTV/Shoutify)                     | [shoutify.app](https://shoutify.app)       |
```
Comment (c-ehrlich) [inline www/src/pages/en/t3-collection.md:14]: https://shoutify.app is a 404

**Comment by KyleTryon** [comment]
```
❯ pnpm format

> @ct3a/root@0.0.0 format ~/Projects/create-t3-app
> turbo format && prettier --write *.{cjs,md,json}

 ERROR  run failed: task `format` not found in turbo `pipeline` in "turbo.json". Are you sure you added it?
 ELIFECYCLE  Command failed with exit code 1.
```

```
❯ pnpm install
Scope: all 3 workspace projects
Lockfile is up to date, resolution step is skipped
Already up to date
```

👀 the turbo.json file > No `format`, but do see `format:check`

```
@ct3a/www:format:check: Checking formatting...
@ct3a/www:format:check: [warn] src/pages/en/t3-collection.md
create-t3-app:format:check: All matched files use Prettier code style!
@ct3a/www:format:check: [warn] Code style issues found in the above file. Forgot to run Prettier?
@ct3a/www:format:check:  ELIFECYCLE  Command failed with exit code 1.
@ct3a/www:format:check: ERROR: command finished with error: command (~/Projects/create-t3-app/www) pnpm run format:check exited (1)
command (~/Projects/create-t3-app/www) pnpm run format:check exited (1)
```

👀 the package.json file > `"format": "turbo format && prettier --write *.{cjs,md,json}",`
`npm run format`

```
❯ npm run format

> @ct3a/root@0.0.0 format
> turbo format && prettier --write *.{cjs,md,json}

 ERROR  run failed: task `format` not found in turbo `pipeline` in "turbo.json". Are you sure you added it?
```

Runs : `prettier --write *.{cjs,md,json}`

```
no matches found: *.cjs
```

runs: 

```
❯ prettier --write *.{md,json}
CONTRIBUTING.md 35ms
README.md 26ms
package-lock.json 109ms
package.json 21ms
tsconfig.eslint.json 3ms
tsconfig.json 5ms
turbo.json 2ms
```

No files changed

File: www/src/pages/en/t3-collection.md (hunk 1 above)
Comment (nexxeln) [reply www/src/pages/en/t3-collection.md:14]: > The link given for the website is currently dead, but we will get a landing page there as soon as possible.

**Comment by juliusmarminge** [comment]
> ```
> ❯ pnpm format
> 
> > @ct3a/root@0.0.0 format ~/Projects/create-t3-app
> > turbo format && prettier --write *.{cjs,md,json}
> 
>  ERROR  run failed: task `format` not found in turbo `pipeline` in "turbo.json". Are you sure you added it?
>  ELIFECYCLE  Command failed with exit code 1.
> ```
> 
> 
>      
>   
> 
> ```
> ❯ pnpm install
> Scope: all 3 workspace projects
> Lockfile is up to date, resolution step is skipped
> Already up to date
> ```
> 
> 
>      
>   
> 
> 👀 the turbo.json file > No `format`, but do see `format:check`
> 
> 
> ```
> @ct3a/www:format:check: Checking formatting...
> @ct3a/www:format:check: [warn] src/pages/en/t3-collection.md
> create-t3-app:format:check: All matched files use Prettier code style!
> @ct3a/www:format:check: [warn] Code style issues found in the above file. Forgot to run Prettier?
> @ct3a/www:format:check:  ELIFECYCLE  Command failed with exit code 1.
> @ct3a/www:format:check: ERROR: command finished with error: command (~/Projects/create-t3-app/www) pnpm run format:check exited (1)
> command (~/Projects/create-t3-app/www) pnpm run format:check exited (1)
> ```
> 
> 
>      
>   
> 
> 👀 the package.json file > `"format": "turbo format && prettier --write *.{cjs,md,json}",`
> `npm run format`
> 
> 
> ```
> ❯ npm run format
> 
> > @ct3a/root@0.0.0 format
> > turbo format && prettier --write *.{cjs,md,json}
> 
>  ERROR  run failed: task `format` not found in turbo `pipeline` in "turbo.json". Are you sure you added it?
> ```
> 
> 
>      
>   
> 
> Runs : `prettier --write *.{cjs,md,json}`
> 
> 
> ```
> no matches found: *.cjs
> ```
> 
> 
>      
>   
> 
> runs:
> 
> 
> ```
> ❯ prettier --write *.{md,json}
> CONTRIBUTING.md 35ms
> README.md 26ms
> package-lock.json 109ms
> package.json 21ms
> tsconfig.eslint.json 3ms
> tsconfig.json 5ms
> turbo.json 2ms
> ```
> 
> 
>      
>   
> 
> No files changed

run it again now, should be fixed

**Comment by juliusmarminge** [comment]
Hey, we just feature-locked main and switched `next` to the default branch. Please resolve any conflicts in this PR that may have occurred from the switch

**Pull request closed without merging**